### PR TITLE
[EPIC-4884] update classic examples to Scanbot SDK 6.0 RC

### PR DESCRIPTION
**Pull request by Den-Rimus**

### CODE REVIEW

File: classic-components-example/adjustable-filters/src/main/java/io/scanbot/example/FilterActivity.kt [hunk 1]
```diff
@@ -4,111 +4,122 @@ import android.app.Activity
...
-            selectedFilter = page.parametricFilters.firstOrNull() ?: LegacyFilter(ImageFilterType.NONE.code)
+        val docId = intent.getStringExtra(DOC_ID_EXTRA)
+        if (docId == null) {
+            this.showToast("Document ID is missing!")
+            finish()
+            return
         }
 
-        binding.filterValue.text = getFilterName()
+        val doc = scanbotSdk.documentApi.loadDocument(docId, createIfNotExist = false)
```
Comment (4BooM04) [inline classic-components-example/adjustable-filters/src/main/java/io/scanbot/example/FilterActivity.kt:64]: should be wrapped in coroutine?

File: classic-components-example/barcode-scanner/src/main/java/io/scanbot/example/ui/BarcodeScanAndCountViewActivity.kt [hunk 2]
```diff
@@ -133,7 +132,7 @@ class BarcodeScanAndCountViewActivity : AppCompatActivity() {
         // TODO to get the result of all scanned barcodes, use the following code
     }
 
-    override fun onResume() {
+    override fun onResume() { // TODO: migrate to Result API to request permissions!
```
Comment (4BooM04) [inline classic-components-example/barcode-scanner/src/main/java/io/scanbot/example/ui/BarcodeScanAndCountViewActivity.kt:135]: should we leave todos on public example?

File: classic-components-example/common/src/main/java/io/scanbot/example/common/ImportImageContract.kt [hunk 3]
```diff
@@ -1,36 +0,0 @@
...
-class ImportImageContract(private val context: Context) : ActivityResultContract<Unit, Bitmap?>() {
-    override fun createIntent(context: Context, input: Unit): Intent {
-        // select an image from photo library and run document detection on it:
-        val imageIntent = Intent()
-        imageIntent.type = "image/*"
-        imageIntent.action = Intent.ACTION_GET_CONTENT
-        imageIntent.putExtra(Intent.EXTRA_LOCAL_ONLY, false)
-        imageIntent.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, false)
-
-       return Intent.createChooser(imageIntent, "Select picture")
```
Comment (4BooM04) [inline classic-components-example/common/src/main/java/io/scanbot/example/common/ImportImageContract.kt:20]: deleted?

File: classic-components-example/pdf-creation/src/main/java/io/scanbot/example/MainActivity.kt [hunk 4]
```diff
@@ -1,170 +1,114 @@
...
-                    imageUris.add(uri)
-                }
-            } ?: intent.data?.let { data ->
-                imageUris.add(data)
+    private suspend fun processDocument(uris: List<Uri>, applyGrayscale: Boolean) {
+        val filters = if (applyGrayscale) listOf(ParametricFilter.grayscaleFilter()) else emptyList()
+        withContext(Dispatchers.Main) { binding.progressBar.visibility = View.VISIBLE }
+
+        val renderedPdfFile = withContext(Dispatchers.Default) {
+            val contourDetector = scanbotSdk.createContourDetector()
```
Comment (4BooM04) [inline classic-components-example/pdf-creation/src/main/java/io/scanbot/example/MainActivity.kt:72]: is this happen in main thresd?

**Review by 4BooM04 — APPROVED** [review]

File: classic-components-example/common/src/main/java/io/scanbot/example/common/ImportImageContract.kt (hunk 3 above)
Comment (Den-Rimus) [reply classic-components-example/common/src/main/java/io/scanbot/example/common/ImportImageContract.kt:20]: Yes. Why?

**Pull request merged**